02-somatic_concepts.Rmd

```diff
@@ -36,29 +36,38 @@ Tumor variants may be hard to detect due to having low allelic fractions and tum
+Small insertions and deletions (indels) are the second most common form of mutation found both in tumors, and occurring as germline variation in the human population. Indels are usually considered to be insertions and deletions of 50 base pairs or less. In protein coding regions, indels can generate in-frame changes to the amino acid structure of a protein when the insertion or deletion is a multiple of three base pairs, resulting in the addition or loss of one amino acid per three base pairs. Indels can also result in frameshift mutations when the insertion or deletion in a coding region is not a multiple of three. These mutations alter the reading frame, resulting in a radical change to the amino acid sequence downstream of the mutation, usually resulting in a non-functional protein product.  
 
+Translocations and fusions are a relatively common variant type found in cancer. Translocations are exchanges of large regions of DNA between different chromosomes, or between different regions of the same chromosome. Translocations may have breakpoints that occur in the middle of two different genes, resulting in fusions that have exons from both genes. A well known fusion results from the Philadelphia Chromosome, first identified in 1973 [@Rowley1973], which forms when chromosome 9 and chromosome 22 break and exchange large regions. This results in a fusion protein consisting of parts of the BCR gene and the ABL1 kinase gene, including the active kinase domain. The presence of BCR causes tetramer formation, which in turn activates the ABL1 kinase in a disregulated manner, driving oncogenic tyrosine kinase signaling. This fusion drives Chronic myeloid leukemia (CML) and is a driver that occurs in other cancer types as well. Targeted therapies against BCR::ABL1, such as imatinib, have been developed as a result of the discovery of this fusion and its driver properties [@Mughal2016]. 
 
+A different type of genomic variants consist of copy number variants (CNVs) or copy neutral loss of heterogeneity (CNLOH). Copy number variants consist of losses or gains of large regions of chromosomes. Losses are usually deletions of genomic regions, and gains are usually two or larger fold duplications of genomic regions. These sorts of variants effect large numbers of genes and play a role in cancer. CNLOH usually results from the replacement of a large stretch of DNA on a chromosome with the same stretch of DNA derived from the homologous chromosome in the same cell. This causes a loss of heterogeneity as all DNA in this region is now identical between both homologous chromosomes, but there is no change in copy number of the genes in this region since there were no losses or gains in the DNA of this region. 
```

Comment:
```suggestion
A different type of genomic variants consists of copy number variants (CNVs) or copy neutral loss of heterogeneity (CNLOH). Copy number variants consist of losses or gains of large regions of chromosomes. Losses are usually deletions of genomic regions, and gains are usually two or larger fold duplications of genomic regions. These sorts of variants effect large numbers of genes and play a role in cancer. CNLOH usually results from the replacement of a large stretch of DNA on a chromosome with the same stretch of DNA derived from the homologous chromosome in the same cell. This causes a loss of heterogeneity as all DNA in this region is now identical between both homologous chromosomes, but there is no change in copy number of the genes in this region since there were no losses or gains in the DNA of this region. 
```
I went with "A different type...consists" in this suggestion, but I could also see deciding to make both plural if we want to count CNVs and CNLOH as two types.